ios/Classes/SwiftFlutterBarcodeScannerPlugin.swift

```diff
@@ -361,15 +374,21 @@ class BarcodeScannerViewController: UIViewController {
         cancelButton.heightAnchor.constraint(equalToConstant: 70.0).isActive = true
         cancelButton.bottomAnchor.constraint(equalTo:view.bottomAnchor,constant: 0).isActive=true
         cancelButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant:10).isActive = true
+        
+        switchCameraButton.translatesAutoresizingMaskIntoConstraints = false
+        // A little bit to the right.
+        switchCameraButton.leftAnchor.constraint(equalTo: view.leftAnchor, constant: 10).isActive = true
+        switchCameraButton.heightAnchor.constraint(equalToConstant: 70.0).isActive = true
+        switchCameraButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 0).isActive = true
     }
     
     /// Flash button click event listener
     @IBAction private func flashButtonClicked() {
```

Comment:
Please handle the flash camera click when the front camera is open and the user clicks the flash icon. Currently the front camera freezes when user clicks flash icon

Reply:
Change pushed.